datafusion/physical-plan/src/aggregates/row_hash.rs

```diff
@@ -29,10 +29,10 @@ use crate::aggregates::{
 };
 use crate::common::IPCWriter;
 use crate::metrics::{BaselineMetrics, RecordOutput};
-use crate::sorts::sort::{read_spill_as_stream, sort_batch};
+use crate::sorts::sort::sort_batch;
```

Comment:
Maybe we can do the same with sort_batch

Reply:
Thanks @alamb I'll probably leave it as is, as it tied to sorting logic. Once we have another use case for making it a helper method and move it the same way for sure.